src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -3242,7 +3242,7 @@ static void on_external_watch_complete_locked(void* arg, grpc_error* error) {
                            "external_connectivity_watcher");
   external_connectivity_watcher_list_remove(w->chand, w);
   gpr_free(w);
-  GRPC_CLOSURE_RUN(follow_up, GRPC_ERROR_REF(error));
+  GRPC_CLOSURE_SCHED(follow_up, GRPC_ERROR_REF(error));
```

Comment:
Why is this being changed?  It looks to me like both the caller and the closure are running in the combiner, so this is the kind of place we want to use run.

Reply:
https://github.com/grpc/grpc/blob/fec4b30d9a94cb6700bfe92ab50a3eccb9ebf128/src/core/ext/filters/client_channel/channel_connectivity.cc#L221

is one place where the closure being run is not initialized for the combiner. 
(This change was actually motivated because of a test failure which led to debugging and changing this.)